test/denormalized_has_many_test.rb

```diff
@@ -90,14 +90,14 @@ def test_cached_associations_after_commit_hook_will_not_fail_on_undefined_parent
 
   def test_cache_without_guessable_inverse_name_raises
     assert_raises IdentityCache::InverseAssociationError do
-      Item.cache_has_many :polymorphic_records, :embed => true
```

Comment:
Looks like this started failing because rails is now smart enough to infer the inverse name.